news/2018-02-18-qat-gazette-february-2018.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "The QAT Gazette: February 2018"
+date: 2018-2-18 09:00:00 +0000
+---
+
+What's been happening in the Beatmap Nominators and Quality Assurance Team, you ask? Mapping successes, exciting current developments and many new team additions all round, that's what! Read on to find out all the details!
+
+![](/wiki/shared/news/banners/qat-blog-logo.png)
+
+Welcome back everyone!
+
+We last brought news to you at the tail-end of 2017, so hopefully you all had an enjoyable Christmas and New Year period. Having skipped over a relatively calm holiday season for the Beatmap Nominators and Quality Assurance Team, January has been a return to normality, and to that effect I'd like to once again bring everyone up to speed on the current state of affairs, as well as give an indication of future plans.
+
+So with that said, let's crack on!
+
+## Within the Beatmap Nominators
+
+Sometimes it can be a drag getting back into the working rhythm, but it seems as though our Beatmap Nominator team didn't even miss a step settling into the new year. Instead, January 2018 saw the **highest count of ranked difficulties and mapsets of any single month in osu! history** - an impressive feat, as well as a promising start for the year I hope! You can find a breakdown of these impressive statistics courtesy of [osu!daily](https://osudaily.net/various.php) if you're interested.
+
+That said, we're always striving towards future growth and even more impressive accomplishments, so here's a little rundown on recent implementations and future plans for the Beatmap Nominators.  
+
+### New Beatmap Nominator promotions
+
+A significant part in the above-mentioned success might be attributed to a few new Nominators joining our ranks - and when I say "a few", I mean twenty-one of them. Yes, really. Without further ado, please welcome the newst additions to the Beatmap Nominators for osu!: 
+
+[- Frontier -](https://osu.ppy.sh/users/4314710), [ailv](https://osu.ppy.sh/users/6933054), [AyanokoRin](https://osu.ppy.sh/users/5376866), [Bubblun](https://osu.ppy.sh/users/6191653), [dsco](https://osu.ppy.sh/users/4330511), [ezek](https://osu.ppy.sh/users/180241), [Hailie](https://osu.ppy.sh/users/5226970), [iYiyo](https://osu.ppy.sh/users/3919785), [Kuron-Kun](https://osu.ppy.sh/users/2697284), [Lanturn](https://osu.ppy.sh/users/1446665), [mancuso_JM_](https://osu.ppy.sh/users/521568), [Namki](https://osu.ppy.sh/users/5248582), [Nevo](https://osu.ppy.sh/users/7451883), [Noffy](https://osu.ppy.sh/users/1541323), [pinataman](https://osu.ppy.sh/users/1235485), [Realazy](https://osu.ppy.sh/users/918297), [Sotarks](https://osu.ppy.sh/users/4452992), [TheKingHenry](https://osu.ppy.sh/users/5128277), [timemon](https://osu.ppy.sh/users/2072005), [UndeadCapulet](https://osu.ppy.sh/users/2523533) and [Wishkey](https://osu.ppy.sh/users/1101704)!
```

Comment:
electoz + gabe is missing

Reply:
I thought 23 sounded familiar but forgot exisitng othermode people. Added them in a new sentence.